test/functional/user_sessions_controller_test.rb

```diff
@@ -266,4 +266,14 @@ class UserSessionsControllerTest < ActionController::TestCase
     post :destroy
     assert_equal "Successfully logged out.",  flash[:notice]
   end
+  
+  test "logging in with banned user through oauth should fail" do
```

Comment:
Well done. Are the tests related to this PR(banned)? If yes, then add it to this pr. If tests are for other omniauth/oauth sequences then add them in a new pr.

Reply:
Yeah they are